src/Agent.Sdk/SecretMasking/LoggedSecretMasker.cs

```diff
@@ -163,5 +163,21 @@ protected virtual void Dispose(bool disposing)
                 _secretMasker = null;
             }
         }
+
+        public void EnableTelemetry()
+        {
+            if (_secretMasker is OssSecretMasker ossMasker)
```

Comment:
OK, so no telemetry for non-OSS secret masking, fine. We intend to zap use of non-OSS masker anyway.

I know the answer to this question, because you didn't code like the following, but tell me what you think of this one-liner?

(_secretMasker as OssSecretMasker)?.EnableTelemetry();